### PR TITLE
Challenge info page

### PR DESCRIPTION
**Pull request by victorbalan**

### CODE REVIEW

File: app/elements/coder/challenge/info/challenge-info-page.html [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+<link rel="import" href="../../../../bower_components/iron-icons/communication-icons.html">
+<link rel="import" href="../../../../bower_components/iron-icons/device-icons.html">
+<dom-module id="challenge-info-page">
+	<style include="iron-flex">
+		:host{
+			display: block;
+			height: 100%;
+		}
+		#wrapper {
+			/*height: calc(100% - 16px);*/
```
Comment (lorenzleutgeb) [inline app/elements/coder/challenge/info/challenge-info-page.html]: ?

File: app/elements/coder/challenge/info/challenge-info-page.html [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+			@apply(--layout-vertical);
+		}
+		#wrapper {
+			@apply(--layout-vertical);
+			padding: 24px;
+		}
+		#infoIcons {
+			/*margin: 0 auto;*/
```
Comment (lorenzleutgeb) [inline app/elements/coder/challenge/info/challenge-info-page.html]: ?

File: app/elements/coder/challenge/info/challenge-info-page.html [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+					case 'COMPLETED':
+						page('/contest/' + this.challenge.id + '/result');
+						break;
+				}
+			},
+			_onStart: function(){
+				page('/contest/' + this.challenge.canonicalName);
+			},
+			_onSecondaryAction: function(){
+
```
Comment (lorenzleutgeb) [inline app/elements/coder/challenge/info/challenge-info-page.html]: ?

File: app/scripts/behaviors/datetime-behavior.js [hunk 4]
```diff
@@ -6,5 +6,42 @@ Behaviors.DateTimeBehavior = {
...
+			time = time + '0';
+		}
+		time = time + hours + ':';
+		if(mins < 10){
+			time = time + '0';
+		}
+		time = time + mins;
+		return time;
+	},
+	_computeDuration: function(duration) {
```
Comment (lorenzleutgeb) [inline app/scripts/behaviors/datetime-behavior.js:30]: Don't we have that code in some other utility already?

File: app/scripts/behaviors/datetime-behavior.js [hunk 5]
```diff
@@ -6,5 +6,42 @@ Behaviors.DateTimeBehavior = {
...
 		}
 		return util.formatDate(date);
-	}
+	},
+	_getDate: function(d){
+		var date = new Date(d * 1000);
+		var result = date.toISOString();
+		return result.substring(0, result.indexOf('T'));
+	},
+	_getTime: function(d){
```
Comment (lorenzleutgeb) [inline app/scripts/behaviors/datetime-behavior.js:15]: Don't we have that code in some other utility already?

File: app/scripts/behaviors/datetime-behavior.js (hunk 4 above)
Comment (victorbalan) [inline app/scripts/behaviors/datetime-behavior.js:30]: we should start migrating to behaviors. Now i added it here and in another pr i`ll replace the util usages

File: app/scripts/behaviors/datetime-behavior.js (hunk 5 above)
Comment (victorbalan) [inline app/scripts/behaviors/datetime-behavior.js:15]: we should start migrating to behaviors. Now i added it here and in another pr i`ll replace the util usages

**Pull request merged**